### PR TITLE
Fixing maven shade plugin for SegmentTarPush so it doesn't include ht…

### PR DESCRIPTION
**Pull request by jenniferdai**
…tpclient

### CODE REVIEW

File: pinot-hadoop/pom.xml [hunk 1]
```diff
@@ -72,8 +72,18 @@
...
+        <exclusion>
+          <groupId>com.linkedin.pinot</groupId>
+          <artifactId>pinot-common</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.linkedin.pinot</groupId>
+      <artifactId>pinot-common</artifactId>
+      <version>0.016</version>
```
Comment (sunithabeeram) [inline pinot-hadoop/pom.xml]: I don't think we should hard code the version.

Comment (jenniferdai) [reply pinot-hadoop/pom.xml]: good call, removed

File: pinot-common/pom.xml [hunk 2]
```diff
@@ -295,6 +295,7 @@
                   <goal>shade</goal>
                 </goals>
                 <configuration>
+                  <shadedArtifactAttached>true</shadedArtifactAttached>
```
Comment (sunithabeeram) [inline pinot-common/pom.xml]: Is this still required if we run with -Pbuild-shaded-jar above?

File: pinot-hadoop/pom.xml [hunk 3]
```diff
@@ -49,6 +49,7 @@
                   <goal>shade</goal>
                 </goals>
                 <configuration>
+                  <shadedArtifactAttached>true</shadedArtifactAttached>
```
Comment (sunithabeeram) [inline pinot-hadoop/pom.xml]: Same question as above.

**Pull request merged**